### ISSUE TITLE
MCP23S17Pin.GPIO_B7 not working

### ISSUE THREAD

**Issue by interdigitec** [Pi4J/pi4j-v1#103]
HI,
I made some test with the new release (1.0-SNAPSHOT).
I try the example MCP23S17GpioExample.java
Durring the program is running, if I change the inputs, I see the changes for the input GPIO_B0 to B6.
If I try to change GPIO_B7 nothing happens and after I cannot change the other GPIO anymore.

BR
Reinhard

**Comment by interdigitec** [comment]
De same with portA: GPIO_A7 is not working

**Comment by savageautomate** [comment]
Confirmed defect.  Fixed in latest 1.0-SNAPSHOT build.

```
 <--Pi4J--> MCP23S17 GPIO Example ... started.
 --> GPIO PIN STATE CHANGE: "MyInput-A0" <GPIO A0> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-A0" <GPIO A0> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-A1" <GPIO A1> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-A1" <GPIO A1> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-A2" <GPIO A2> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-A2" <GPIO A2> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-A3" <GPIO A3> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-A3" <GPIO A3> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-A4" <GPIO A4> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-A4" <GPIO A4> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-A5" <GPIO A5> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-A5" <GPIO A5> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-A6" <GPIO A6> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-A6" <GPIO A6> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-A7" <GPIO A7> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-A7" <GPIO A7> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-B0" <GPIO B0> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-B0" <GPIO B0> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-B1" <GPIO B1> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-B1" <GPIO B1> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-B3" <GPIO B2> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-B3" <GPIO B2> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-B3" <GPIO B3> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-B3" <GPIO B3> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-B4" <GPIO B4> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-B4" <GPIO B4> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-B5" <GPIO B5> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-B5" <GPIO B5> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-B6" <GPIO B6> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-B6" <GPIO B6> = HIGH
 --> GPIO PIN STATE CHANGE: "MyInput-B7" <GPIO B7> = LOW
 --> GPIO PIN STATE CHANGE: "MyInput-B7" <GPIO B7> = HIGH
```
